warehouse/templates/includes/file-details.html

```diff
@@ -118,26 +138,67 @@ <h3>Provenance</h3>
+          {{ publisher(bundle.publisher, bundle.attestations[0].certificate_claims )}}
           Attestations:
+          <small>
+            <i>Values shown here reflect the state when the release was signed and may no longer be current.</i>
+          </small>
           <ul>
-            {% for attestation in bundle.attestations %}
+          {% for attestation in bundle.attestations %}
             <li>
               {% set statement = attestation.statement %}
-              Statement type: <a href="{{ statement._type }}"><code>{{ statement._type }}</code></a>
+              Statement:
               <ul>
+                <li>Statement type: <a href="{{ statement._type }}"><code>{{ statement._type }}</code></a></li>
                 <li>Predicate type: <a href="{{ statement.predicateType }}"><code>{{ statement.predicateType }}</code></a></li>
                 <li>Subject name: <code>{{ statement.subject[0].name }}</code></li>
                 <li>Subject digest: <code>{{ statement.subject[0].digest.sha256 }}</code></li>
                 {% set tlog_entry = attestation.verification_material.transparency_entries[0] %}
                 <li>Sigstore transparency entry: <a href="https://search.sigstore.dev/?logIndex={{ tlog_entry.logIndex }}">{{ tlog_entry.logIndex }}</a> </li>
                 <li>Sigstore integration time: {{ humanize(tlog_entry.integratedTime | int | ctime, time="true") }} </li>
               </ul>
-            </li>
+              {% set claims = attestation.certificate_claims %}
+              Source repository:
+              <ul>
+                <!-- 1.3.6.1.4.1.57264.1.12 | Source Repository URI-->
+                <!-- 1.3.6.1.4.1.57264.1.13 | Source Repository Digest-->
+                <li>Permalink: <a href="{{ repo_url_with_reference(claims.get("1.3.6.1.4.1.57264.1.12"), claims.get("1.3.6.1.4.1.57264.1.13")) }}">
+                    <code>{{ repository_name(bundle.publisher) }}#{{ claims.get("1.3.6.1.4.1.57264.1.13") }}</code>
+                </a></li>
+                <!-- 1.3.6.1.4.1.57264.1.14 | Source Repository Ref-->
+                <li>Branch / Tag: <a href="{{ repo_url_with_reference(claims.get("1.3.6.1.4.1.57264.1.12"), claims.get("1.3.6.1.4.1.57264.1.14")) }} ">
+                    <code>{{ claims.get("1.3.6.1.4.1.57264.1.14") }}</code>
+                </a></li>
+                <!-- 1.3.6.1.4.1.57264.1.15 | Source Repository Identifier-->
+                <!-- 1.3.6.1.4.1.57264.1.16 | Source Repository Owner URI-->
+                <li>Owner: <a href="{{ claims.get("1.3.6.1.4.1.57264.1.16") }}">{{ claims.get("1.3.6.1.4.1.57264.1.16") }}</a></li>
+                <!-- 1.3.6.1.4.1.57264.1.17 | Source Repository Owner Identifier-->
+                <!-- 1.3.6.1.4.1.57264.1.22 | Source Repository Visibility At Signing-->
+                <li>Access: <code>{{ claims.get("1.3.6.1.4.1.57264.1.22") }}</code></li>
+              </ul>
+              Build information:
```

Comment:
"Build information" for a publish attestation is confusing, we shouldn't really be using the word "build" in the context of a "publish" attestation I think.

Reply:
I've renamed it `Publication detail,` but leave this conversation open if someone finds better options.